graphing/src/templates/experiment.html

```diff
@@ -20,6 +20,8 @@ <h3>{{experiment}}</h3>
 	<a href="{% url 'home' %}{{experiment}}/{{x}}" class="btn btn-default btn">{{x}}</a>
 	{% endfor %}
 
+    <a href="{% url 'home' %}{{experiment}}/dilutions" class="btn btn-default btn">dil</a>
```

Comment:
Can you make this say "Dilutions/Media Consumption" instead of just "dil"? I think that will look better and be more clear.